### PR TITLE
make test pilot reasonably responsive

### LINKED ISSUE #241: Add more media queries for smaller devices
Currently you need a browser that is at least 1024px wide to use the site (or at least sign up).
My browser was about 600px wide, and clicking the "Get started with your Firefox Account" button launched a helicopter based modal dialog which was offscreen.

If I resize to 950px wide, I can see the helicopter message.
If I resize to ~1000px wide, I can also see the Mozilla Tabzilla tab.

idea_town_and_launch_java_web_start_slave_agent_via_windows_scheduler_-_jenkins_-_jenkins_wiki_and_microsoft_remote_desktop

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #241

### CODE REVIEW

File: testpilot/frontend/static-src/styles/modules/_experiment-details.scss [hunk 1]
```diff
@@ -109,25 +136,30 @@
...
+    margin: 0;
+  }
+
+  .name {
+    margin: 0;
   }
 
   .contributor {
-    padding-left: 10px;
+    padding-left: 1$grid-unit * .5;
```
Comment (pdehaan) [inline testpilot/frontend/static-src/styles/modules/_experiment-details.scss]: Should this have the leading `1`?

Comment (johngruen) [inline testpilot/frontend/static-src/styles/modules/_experiment-details.scss]: @pdehaan <3

**Comment by pdehaan** [comment]
@johngruen Great work, as usual! This looks beautiful.

I did notice that on the /experiments page, you lose the [...] button completely at around 710px (it looks like it stops pinning to right edge at around 770px)
firefox_test_pilot_-_experiments

I also apparently forgot to add any fixture data, so my UI looks pretty barren on /experiments. Not sure if we need to style an empty state for this, or if we're always reasonably sure that we'll have at least 1 active experiment.

firefox_test_pilot_-_experiments

**Comment by pdehaan** [comment]
With the default fixture data loaded, it looks like the avatars+names are _super_ tight on the experiments details pages:

firefox_test_pilot_-_activity_stream

Also slightly weird that we see headings for **Version** and **Contribute** even though those have no value.

---

FWIW, here's what it currently looks like on testpilot.dev.mozaws:

firefox_test_pilot_-_activity_stream

**Comment by pdehaan** [comment]
Possibly also worth noting is that the modal popups need seemingly at least 570px of vertical space. If you shrink your browser to less than 470px, you lose the "Cancel" link/button completely and cannot scroll the modal to see the header or contents (it scrolls the page in the background).

firefox_test_pilot_-_universal_search

**Comment by pdehaan** [comment]
Apart from that, I didn't see any other issues.
Most of my other nits are unrelated and I can file separately so we can ignore those on a per-issue basis.

I'll let @meandavejustice and @lmorchard give the final :eyes: and :+1:/:-1:.

**Comment by johngruen** [comment]
@pdehaan:
- i fixed the broken header settings thing
- go ahead and file bugs for
  - empty state
  - squashed contributors
  - tall modals

@lmorchard @meandavejustice review and merge whenevz (hopefully soon)

**Comment by lmorchard** [comment]
FWIW, the empty fields thing isn't new with this PR, I can see it happening on master branch on dev. Also, those feedback dialogs are on the way out, so we can fix that in #567

**Pull request merged**